src/main/cpp/controllers/turret/ShooterSaveState.h

```diff
@@ -49,7 +48,7 @@ class ShooterSaveState : public IState
     double degreesToTurn;
 
     private:
-
+    ControlData*   m_controlData;

```

Comment:
why 2 variables for ControlData?  Isn't the previous version of the file correct?